### PR TITLE
Replaced Fontawesome Icons with Iconify Icons

### PR DESCRIPTION
**Pull request by 1hanzla100**
### Replaced Fontawesome Icons with Iconify Icons
To Find Icons **Visit ** https://iconify.design/icon-sets/

### CODE REVIEW

**Comment by naveen521kk** [comment]
Why though? Font Awesome is more famous and people know it more,  IMHO, it would be better to have Font awesome, but I am open to suggestions.

**Comment by 1hanzla100** [comment]
> Why though? Font Awesome is more famous and people know it more, IMHO, it would be better to have Font awesome, but I am open to suggestions.

Fontawesome have limited variety of . On the other side, iconify have large variety of icons and the look much better than fontawesome icons

**Comment by saadpasta** [comment]
@naveen521kk  I think that colorful icons are looking awesome what do you say. 
But I think that it consists of a lot of icons, @1hanzla100 it is possible we can get only the icons we need or only tech-related icons?

**Comment by 1hanzla100** [comment]
> @naveen521kk I think that colorful icons are looking awesome what do you say.
> But I think that it consists of a lot of icons, @1hanzla100 it is possible we can get only the icons we need or only tech-related icons?

ok i will remove extra icons

**Comment by saadpasta** [comment]
@1hanzla100 How do you plan to do that ?

**Comment by 1hanzla100** [comment]
> @1hanzla100 How do you plan to do that ?

Read official iconify documentation

**Comment by saadpasta** [comment]
@1hanzla100 Sorry for not clearing myself. When the script loads it get all the icons. I want to know if there is a way to get only tech-related icons. I didn't mean to remove extra skills.  
Thanks

**Comment by 1hanzla100** [comment]
> @1hanzla100 Sorry for not clearing myself. When the script loads it get all the icons. I want to know if there is a way to get only tech-related icons. I didn't mean to remove extra skills.
> Thanks

I cannot understand what do mean??

**Comment by 1hanzla100** [comment]
You are saying that using fontawesome and iconify side by side??

**Comment by 1hanzla100** [comment]
> @naveen521kk I think that colorful icons are looking awesome what do you say.
> But I think that it consists of a lot of icons, @1hanzla100 it is possible we can get only the icons we need or only tech-related icons?

iconnify works exactly like fontawesome. If Want to add or modify icons just we have to change class names like in fontawesome.

**Comment by 1hanzla100** [comment]
@saadpasta  Kindly Approve My Pull Request to master

**Comment by naveen521kk** [comment]
> @naveen521kk I think that colorful icons are looking awesome what do you say.

No. I don't like the *colorful* icons though. At least, that doesn't look good when things are changing from dark to light mode or vice versa. Current setup looks beautiful to me.

**Comment by 1hanzla100** [comment]
> > @naveen521kk I think that colorful icons are looking awesome what do you say.
> 
> No. I don't like the _colorful_ icons though. At least, that doesn't look good when things are changing from dark to light mode or vice versa. Current setup looks beautiful to me. But again, I am open to suggestions.

@naveen521kk I think they are looking better in both dark and light mode 
icons

**Comment by saadpasta** [comment]
@naveen521kk let's discuss with @kartikcho what do you think?

**Comment by kartikcho** [comment]
Thanks for updating the icons @1hanzla100, I like them but this feels more of a personal preference than a feature that could be **objectively** better than the old icons. 

That being said, although I don't like fa icons personally, the current on hover effect adds a slight amount of interactivity in the portfolio which could be a nice add on for a lot of users. If they feel like they could change something to make it better, they definitely should go for it!

This also includes another dependency on top of fa as fa would still be used for the socials link icons.

So TLDR, this is more a personal preference and not exactly a feature PR. I hate to say no to a PR but if you want to help out the project, take a look at the issues board! 

If you still want to improve on the icons, maybe add some interactivity with your icons in this PR

**Comment by 1hanzla100** [comment]
@kartikcho I have Added Hover affects in skill icons. Check the out.

File: README.md [hunk 1]
```diff
@@ -270,7 +268,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
     <td align="center"><a href="https://bharatkammakatla.github.io"><img src="https://avatars1.githubusercontent.com/u/28840761?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Bharat Kammakatla</b></sub></a><br /><a href="#design-BharatKammakatla" title="Design">🎨</a></td>
-    <td align="center"><a href="http://bit.ly/garimasingh"><img src="https://avatars2.githubusercontent.com/u/44302373?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Garima Singh</b></sub></a><br /><a href="https://github.com/saadpasta/developerFolio/commits?author=garimasingh128" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/1hanzla100"><img src="https://avatars3.githubusercontent.com/u/59178380?v=4" width="100px;" alt=""/><br /><sub><b>Hanzla Tauqeer</b></sub></a><br /><a href="https://github.com/saadpasta/developerFolio/commits?author=BourgeoisBenjamin" title="Code">💻</a></td>
```
Comment (kartikcho) [inline README.md:271]: You don't need to update this, we add a contributor through a bot.

File: README.md [hunk 2]
```diff
@@ -296,6 +294,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
     <td align="center"><a href="https://tamojitdas.netlify.app"><img src="https://avatars0.githubusercontent.com/u/40804626?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Tamojit Das</b></sub></a><br /><a href="https://github.com/saadpasta/developerFolio/commits?author=tamojit-123" title="Documentation">📖</a></td>
     <td align="center"><a href="https://warengonzaga.com"><img src="https://avatars1.githubusercontent.com/u/15052701?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Waren Gonzaga</b></sub></a><br /><a href="https://github.com/saadpasta/developerFolio/commits?author=WarenGonzaga" title="Code">💻</a></td>
+    <td align="center"><a href="http://bit.ly/garimasingh"><img src="https://avatars2.githubusercontent.com/u/44302373?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Garima Singh</b></sub></a><br /><a href="https://github.com/saadpasta/developerFolio/commits?author=garimasingh128" title="Code">💻</a></td>
```
Comment (kartikcho) [inline README.md:297]: Same here as above.

File: src/components/softwareSkills/SoftwareSkill.css [hunk 3]
```diff
@@ -24,6 +22,20 @@
...
   font-size: 10px;
 }
-.software-skill-inline > i:hover ~ p{
-  color: #645beb;
+.software-skill-inline:hover{
+  transform: scale(0.95);
+  transition: .4s;
+}
+
+.iconify{
```
Comment (kartikcho) [inline src/components/softwareSkills/SoftwareSkill.css:30]: This class name could be changed to something more descriptive or could just use `.software-skill-inline > span`

File: src/components/softwareSkills/SoftwareSkill.js [hunk 4]
```diff
@@ -10,7 +10,12 @@ export default function SoftwareSkill() {
           {skillsSection.softwareSkills.map((skills,i) => {
             return (
               <li key={i} className="software-skill-inline" name={skills.skillName}>
-                <i className={skills.fontAwesomeClassname}></i>
+                <span
+                      className="iconify"
+                      data-icon={skills.fontAwesomeClassname}
```
Comment (kartikcho) [inline src/components/softwareSkills/SoftwareSkill.js:15]: the `fontAwesomeClassname` should be updated to use iconifyClassname or something similar

**Review by kartikcho — CHANGES_REQUESTED** [review]
I'm still not exactly 'for' this addition as I think font awesome is something almost every web dev is familiar with and this could require them to read through more docs (although the big library of icons is nice).

Also, the hover animation is neomorphic but the portfolio follows material design loosely so we could maybe add a more minimal hover effect, maybe something using css filters?

File: README.md (hunk 1 above)
Comment (1hanzla100) [reply README.md]: i have reverted it back

File: src/components/softwareSkills/SoftwareSkill.js (hunk 4 above)
Comment (1hanzla100) [reply src/components/softwareSkills/SoftwareSkill.js]: @kartikcho 
i Have Changed Hover Effect  | Renamed fontawesomeClassname to iconfyClassname and Reverted Edits in REadme.md

**Comment by 1hanzla100** [comment]
Closed PR
